docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -28,57 +26,93 @@ work with Teleport.
   [guides](../enroll-resources/database-access/guides/guides.mdx) for how to set up the Teleport
   Database Service for your database.
 
-### Get connection information
+## How GUI clients access Teleport-protected databases
+
+In a typical setup, clients accessing a database protected by Teleport send
+traffic to the Teleport Proxy Service in the database's native protocol, and the
+Proxy Service forwards the traffic to and from the protected database. End users
+use TLS certificates to authenticate with Teleport-protected databases. 
+
+GUI clients need to present a certificate to the Teleport Database Service and
+check the database server certificate issued by a Teleport-protected database.
+There are three ways to do this:
+
+- **Authenticated tunnel (recommended):** The local proxy handles loading the
+  Teleport user's client certificate and checking the certificate issued by the
+  Teleport-protected database. The connection remains authenticated, and clients
+  only need to forward database protocol messages to it.
+- **Unauthenticated local proxy:** The local GUI client needs to load the
+  Teleport user's certificate and check database certificates. This requires
+  configuring the client to access TLS credentials, including the user
+  certificate and private key, and a certificate issued by the database's
+  certificate authority. 
+- **Using a remote host and port:** For self-hosted clusters with TLS
+  multiplexing disabled on the Teleport Proxy Service, you can configure GUI
+  clients to communicate with a port on the Proxy Service reserved for
+  traffic in the protected database's protocol. We recommend TLS multiplexing
+  for a typical Teleport cluster, and it is enabled by default on Teleport
+  Cloud.
+
+To access a Teleport-protected database with a GUI client, you will need to
+retrieve connection information to pass to your client.
+
+Determine the database to access by listing databases you can connect to:
 
-Get information about the host and port where your database is available so you
-can configure your GUI client to access the database.
+```code
+$ tsh db ls
+# Name
+# -------------------
+# database-name
+```
 
-#### Using a local proxy server
+Replace <Var name="database-name" /> with the name of the database you want to
+connect to.
 
-Use the `tsh proxy db` command to start a local TLS proxy your GUI database
-client will be connecting to. This command requires that you use Teleport
-Enterprise (Cloud) or, if self-hosting Teleport, have enabled [TLS
-routing](../admin-guides/management/operations/tls-routing.mdx) mode.
+### Authenticated tunnel
```

Comment:
What do you think about including a link to https://goteleport.com/docs/connect-your-client/teleport-connect/#connecting-to-a-database in this section? Teleport Connect is not mentioned in this page at all, but at the moment databases are the most commonly used resources in Connect.